Exam1/Models/Data/DataListModels.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Exam1.Models.Base;
+using Exam1.Models.Data.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Exam1.Models.Data
+{
+    class DataListModels
+    {
+        public List<IManager> managers;

```

Comment:
Bad convention: you have two options
- Make`managers` `private`
- Rename `managers` to `Managers` with `set`, `get`